packages/indexer-agent/src/query-fees/allocations.ts

```diff
@@ -194,14 +203,104 @@ export class AllocationReceiptCollector implements ReceiptCollector {
   private startVoucherProcessing() {
     timer(30_000).pipe(async () => {
       const pendingVouchers = await this.pendingVouchers()
-      for (const voucher of pendingVouchers) {
-        await this.submitVoucher(voucher)
+
+      const logger = this.logger.child({})
+
+      const vouchers = await pReduce(
+        pendingVouchers,
+        async (results, voucher) => {
+          if (
+            await this.allocationExchange.allocationsRedeemed(
+              voucher.allocation,
+            )
+          ) {
+            logger.warn(
+              `Query fee voucher for allocation already redeemed, deleted local voucher copy`,
```

Comment:
This log doesn't necessarily imply a successful deletion; maybe update `deleted` --> `deleting`?  Or move log to after the `destroy` operation.

Reply:
Will do